qgis-app/styles/views.py

```diff
@@ -379,13 +387,27 @@ def style_download(request, pk):
     else:
         style.increase_download_counter()
         style.save()
-    with open(style.xml_file.file.name, 'rb') as style_file:
-        file_content = style_file.read()
-        response = HttpResponse(file_content, content_type='application/xml')
-        response['Content-Disposition'] = 'attachment; filename=%s.xml' % (
-            style.name
-        )
-        return response
+
+    # zip the style and license.txt
+    filenames = (style.xml_file.file.name, LICENSE_FILE)
```

Comment:
@sumandari I think it would be better if you create a function for this, maybe put it in util class. So both view can just call the function.

Reply:
Thanks @dimasciput. I created a function per your advice.